### PR TITLE
Add Stella Glow

### PR DESCRIPTION
**Pull request by rato123**
Adds Stella Glow.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/stella-glow/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Stella Glow"
+description = "Stella Glow is a turn-based role-playing game developed by Imageepoch for the Nintendo 3DS. The game was released in Japan by Sega in June 2015, in North America by Atlus in November 2015, and in Europe and Australia by NIS America in March 2016. The game's story centers around a young man who must journey to unite four witches so he can save his home town from destruction. It is the last game developed by Imageepoch. (from Wikipedia)"
```
Comment (TheKoopaKingdom) [inline games/stella-glow/game.dat]: Having the release dates in here is redundant.

Comment (TheKoopaKingdom) [inline games/stella-glow/game.dat]: The `(from Wikipedia)` isn't needed here.

File: games/stella-glow/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+region = "usa"
+release_date = "2015-11-17"
+
+[[testcases]]
+title = "0004000000173700"
+compatibility = "1"
+date = "2017-11-18"
+version = "HEAD-de0af83"
+cpu = "Intel Core i7-4510U"
+gpu = "GeForce GT 740M"
```
Comment (TheKoopaKingdom) [inline games/stella-glow/game.dat]: Prepend `Nvidia` to this.

**Comment by Leo626** [comment]
LGTM pending changes.

File: games/stella-glow/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+os = "Windows 10 1709"
```
Comment (Hexagon12) [inline games/stella-glow/game.dat]: That's the wrong Windows version. It should be _Windows 10 16299_.

**Comment by Leo626** [comment]
Please address the comments or this will be closed.

**Pull request closed without merging**